backend/routes/indexer/route.go

```diff
@@ -332,6 +334,41 @@ func StartMessageProcessor() {
 	}()
 }
 
+func AddVotableColor(hex string, dayIndex int) error {
+	dbHost := os.Getenv("DB_HOST")
```

Comment:
I'd recommend looking at how other indexed events are setup. Like this one https://github.com/keep-starknet-strange/art-peace/blob/bdfef62019ede21e92dd169037aa3bb328c1e350/backend/routes/indexer/vote.go#L10

There are already modules to make interacting with postgres easier for example

You'll also need to add the event setup to the indexer directory, here : https://github.com/keep-starknet-strange/art-peace/blob/main/indexer/script.js